scripts/minikube-minishift-setup-env.sh

```diff
@@ -6,6 +6,8 @@ shout() {
   set -x
 }
 
+shout "Setting up environment in `hostname`"
```

Comment:
I would not want to expose this information on prow side. Jenkins should already be aware of the hostname